### ISSUE TITLE
hidden_dims['dim_ffy'] is never passed to XEyTransformerLayer (dropout/layer_norm_eps too) — intentional?

### ISSUE THREAD

**Issue by ArslanArdavic** [cvignac/DiGress#114]
Hi, and thanks for releasing DiGress — I'm building on it and ran into something small that I wanted to check rather than assume is a bug.

`GraphTransformer.__init__` constructs its layers with six keyword arguments ([`src/models/transformer_model.py#L238-L243`](https://github.com/cvignac/DiGress/blob/main/src/models/transformer_model.py#L238-L243)):

```python
self.tf_layers = nn.ModuleList([XEyTransformerLayer(dx=hidden_dims['dx'],
                                                    de=hidden_dims['de'],
                                                    dy=hidden_dims['dy'],
                                                    n_head=hidden_dims['n_head'],
                                                    dim_ffX=hidden_dims['dim_ffX'],
                                                    dim_ffE=hidden_dims['dim_ffE'])
                                for i in range(n_layers)])
```

`dim_ffy`, `dropout` and `layer_norm_eps` are not forwarded, so they keep the `XEyTransformerLayer` defaults at [L26-L28](https://github.com/cvignac/DiGress/blob/main/src/models/transformer_model.py#L26-L28) — `dim_ffy=2048`, `dropout=0.1`, `layer_norm_eps=1e-5`. Since `hidden_dims` is a plain dict, an unused key raises nothing.

The reason I noticed is that every shipped config declares `dim_ffy`:

| config | declared `dim_ffy` | actually built |
| --- | --- | --- |
| `configs/model/discrete.yaml`, `continuous.yaml` | 128 | 2048 |
| `configs/experiment/{sbm,planar,moses,guacamol}.yaml` | 256 | 2048 |
| `configs/experiment/debug.yaml` | 25 | 2048 |
| `configs/experiment/{comm20,qm9_no_h,qm9_with_h,nx_graphs,test}.yaml` | 128 (inherited) | 2048 |

Building the model from `discrete.yaml`-style dims (`n_layers=6, dx=256, de=64, dy=128, dim_ffX=256, dim_ffE=128, dim_ffy=256`) gives:

```
tf_layers[0].lin_y1.weight   torch.Size([2048, 128])     # requested 256
total params                 8.210 M
  of which lin_y1/lin_y2     3.159 M   (38%)
```

So ~38% of the parameters sit in the feed-forward block over the graph-level vector `y`, which carries `t` plus the extra features. Honouring `dim_ffy=256` would give 5.446 M instead.

My questions:

1. Is the wide `y` FFN intentional (i.e. `dim_ffy=2048` is the value the reported models were trained with, and the config entries are vestigial), or is this an oversight and `dim_ffy` was meant to be forwarded?
2. Same question for `dropout` — with no constructor path, `model.eval()` is the only way to turn it off, though the docstring at L23 says "0 to disable".

I ask because it affects how one reproduces the released numbers: passing `dim_ffy` through would change the parameter count of every released configuration, so I'd rather not "fix" it silently in derived work if 2048 is what you trained with. Happy to send a small PR either way — forwarding the three arguments, or adding a comment noting that `dim_ffy` in the configs is not used.
